docs/runbooks/mercury-payment-executor.md

```
@@ -0,0 +1,219 @@
+FROM cc_actions_log
+WHERE action_type = 'payment'
+  AND intent_id IS NULL
+ORDER BY executed_at DESC
+LIMIT 50;
+```
+
+## Idempotency
+
+The dispatcher computes a deterministic idempotency key per attempt:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)